test/performance/io/format_vienna_benchmark.cpp

```diff
@@ -0,0 +1,151 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#if __has_include(<seqan/rna_io.h>)
+    #include <seqan/rna_io.h>
+#endif
+
+#include <sstream>
+#include <string>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/io/structure_file/input.hpp>
+#include <seqan3/io/structure_file/input_format_concept.hpp>
+#include <seqan3/io/structure_file/output.hpp>
+#include <seqan3/io/structure_file/output_format_concept.hpp>
```

Comment:
do you need these headers?

Reply:
I need the input/output header, but not the concept header. I remove the latter.